scripts/generate-artifacthub-repo.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cat <<EOF >"artifacthub-repo.yml"
+name: "skupper"
```

Comment:
@fgiorgetti do you think that this artifacthub-repo.yml can collide with the existing one created for the operator? let me know if I should change something.

Reply:
@nluaces Is this still a valid question? I am not seeing this file anymore.

Reply:
Yes, it is. The file has been renamed to `scripts/push-artifacthub-repo.sh`

Reply:
For the operator, we push two images, skupper-operator-index and skupper-operator-bundle.
IIUC this script is pushing to quay.io/skupper/helm. Unless there is some change in V2 that I
am not yet aware, I believe it should not conflict.